.github/workflows/monodocs_build.yml

```diff
@@ -1,7 +1,7 @@
 name: Monodocs Build
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
```

Comment:
why are we removing this?

Reply:
It allows the GA workflow to run against all pushed commits without being canceled when a new PR is merged

Reply:
Makes sense. We use this pattern in a [bunch of our repos](https://github.com/search?q=org%3Aflyteorg+github.event.pull_request.number+%7C%7C+github.ref+org&type=code), can you also make sure these are changed?